recipes-kernel/linux/linux-bubblegum_3.10.bb

```diff
@@ -0,0 +1,71 @@
+require linux.inc
+
+DESCRIPTION = "Generic 96boards kernel"
```

Comment:
probably need a better description. this is definitely not a generic 96boards kernel..